File: CollegeABCApp/CollegeABCApp/Model/StudentModel.cs
using System.Runtime.Serialization;

namespace CollegeABCApp.Model
{
    [DataContract]
    public class StudentModel
    {
        [DataMember(Name = "Id")]
        public string Id { get; set; }

        [DataMember(Name = "StudentName")]
        public string StudentName { get; set; }

        [DataMember(Name = "Middlename")]
        public string Middlename { get; set; }

        [DataMember(Name = "Lastname")]
        public string Lastname { get; set; }

        [DataMember(Name = "Phone")]
        public string Phone { get; set; }

        [DataMember(Name = "Gender")]
        public string Gender { get; set; }

        [DataMember(Name = "Address")]
        public string Address { get; set; }

      


        [DataMember(Name = "Cityid")]
        public int Cityid { get; set; }

        [DataMember(Name = "StateId")]
        public int StateId { get; set; }

        [DataMember(Name = "CountryId")]
        public int CountryId { get; set; }


        [DataMember(Name = "Zipcode")]
        public int Zipcode { get; set; }
    }
}

File: AngularCollegeApp/src/app/models/student.model.ts
export class Student {
    Id: string;
    StudentName: string;
    Middlename: string;
    Lastname: string;
    Phone: string;
    Gender: string;
    Address: string;
    Country: string;
    State: string;
    City: string;
    Cityid: string;
    StateId: string;
    CountryId: string;
    Zipcode: string;
}
export class Country {
    CountryId: string;
    CountryName: string;
  }
  
  export class State {
    StateId: string;
    StateName: string;
    CountryId: string;
  }
  
  export class City {
    Cityid: string;
    CityName: string;
    StateId: string;
    CountryId: string;
  }

  
export interface DialogData {
  Id: string;
}